src/scripts/main.js

```diff
@@ -1,3 +1,37 @@
 'use strict';
 
-// write code here
+const listSalary = [...document.querySelectorAll('li')];
```

Comment:
It's better to use a more descriptive variable name instead of 'listSalary'. For example, 'salaryList' or 'employeeSalaries' would be more clear.